tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_AK_3km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR.yaml

```diff
@@ -6,16 +6,16 @@ metadata:
 user:
```

Comment:
Should the description be updated to note that the physics suite being used is HRRR?